### DISCUSSION TITLE
Keeping page bundle resources outside of the bundle directory?

### DISCUSSION THREAD

**Discussion by dky** [nunocoracao/blowfish#783] [Q&A] [answered]
Hello, for `https://blowfish.page/docs/thumbnails,` is there any way to host the featured.png outside the content directory? 

I can imagine a situation where you'd have hundreds of images that you wouldn't want to check into Git, as the CI process syncing would take very long. Ideally, I want to only text inside my project repo and have all pictures located externally via Fastly or S3. This way, I'm not storing hundreds of images inside the same git repo.

Here's an example:

```md
new-post
	index.md
	feature.png (can this live somewhere potentially in static or some other directory)?
```

This way, I could manage the directory independently via another mechanism. I've been digging around and found this:

https://discourse.gohugo.io/t/placing-per-post-image-assets-separately-from-content/27246

But unsure how this would work with page bundles. Any help is appreciated!

Thanks for all the hard work on the theme; it's brilliant!

**Comment by nunocoracao** [comment, marked as answer]
@dky thanks for this sounds like an excellent addition. Multiple users have been asking for ways to have remote image support across different features within Blowfish. Feel free to open the feature request so that I can prioritize it accordingly if you know how also happy to review a PR.
